asmcomp/power/proc.ml

```diff
@@ -264,7 +267,7 @@ let safe_register_pressure = function
 
 let max_register_pressure = function
     Iextcall _ -> [| 13; 18 |]
-  | _ -> [| 23; 30 |]
+  | _ -> [| 23; 32 |]
```

Comment:
Was this `30` incorrect before, as there has only been one register added to the allocation pool in this PR? (Just asking for my own understanding)

Reply:
Right.  30 (before) / 31 (now) is conservative but correct, as it accounts for the temporary register used for intoffloat and for store float32.  32 (now) does not account for that, my mistake.  I just pushed a more explicit version of this function that distinguishes between the intoffloat/store-float32 cases and the other instructions.